docs/docs/etl-pipeline-tutorial/3-data-quality.md

```diff
@@ -51,16 +51,16 @@ The asset check is using the same `DuckDBResource` resource we defined for the a
 
 With the Dagster UI, you can now see that an asset check is associated with the `joined_data` asset.
 
-TODO: Screenshot
+![2048 resolution](/images/tutorial/etl-tutorial/asset-check.png)
 
 Asset checks will run when an asset is materialized, but asset checks can also be executed manually in the UI:
 
 1. Reload your Definitions.
```

Comment:
I think we will need to tell people to run `dg dev` again and navigate to `127.0.0.1:3000` here